Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Zan'atali holy sites revision

## Changelog:
- Moved one of Zan'atali holy sites from Nazwatha to Mount Kajaro

## Developer changelog:
- Changed existing modifier for Mount Kajaro HS to +2 learning, as a nod to Kaja'mite.

## Tests:

# How to test:
- Play as Zan'atali character and check if Mount Kajaro holy site is set properly and gives +2 learning
